### ISSUE TITLE
Soundcloud widgets are not reliably replaced

### ISSUE THREAD

**Issue by tjhietala** [EFForg/privacybadger#2331]
Soundcloud widget replacement does not seem to be working reliably.

Works here: https://soundcloud.com/pages/embed

Fails here (as an example): https://www.kvraudio.com/forum/viewtopic.php?f=14&t=521921&sid=e014f2dd651d6e75b9021d9265b2d7ef

Possibly related to #787

**Comment by ghostwords** [comment]
Hello! Thanks for the report.

Looks like this forum uses a custom loading solution for (SoundCloud) embeds. I think the page includes JavaScript that looks for SoundCloud links and appends the appropriate embed to each. The problem is that our replacement code runs before this replacement happens and doesn't run again so it misses the widget. (The replacement seems broken both when `soundcloud.com` is cookieblocked and when it is blocked.)

Basically our widget replacement doesn't yet work with custom embed loaders (https://github.com/EFForg/privacybadger/pull/2262#issuecomment-451493405).

**Comment by ghostwords** [comment]
This one is replaced, sort of, but fails to activate on click: https://postlight.com/trackchanges/podcast/accountability-tools:-paul-and-rich-on-how-they-stay-on-track
